foundations/javascript_basics/clean_code.md

```diff
@@ -148,7 +148,7 @@ You might wonder why this variable is declared with all caps when we recommended
 
 ### Indentation and line length
 
-Now it's time to head to more controversial topics. The war between coders that use tabs and coders that use spaces to indent their code is essentially a joke by now, as demonstrated in the [tabs versus spaces scene from the show Silicon Valley](https://www.youtube.com/watch?v=SsoOG6ZeyUI).
+Now it's time to head to more controversial topics. There's already a joke about the [war between coders who indent with spaces versus tabs](https://www.reddit.com/r/programming/comments/p1j1c/tabs_vs_spaces_vs_both/).
```

Comment:
On a second read, what do you think about the below? Sorry to be a hassle xD Just feel it needs more focus on things being light-hearted, which helps reinforce the "conventions, not absolutes" tone of the lesson.

```suggestion
Now it's time to head to more "controversial" topics (there's a joke about the [war between coders who indent with spaces versus tabs](https://www.reddit.com/r/programming/comments/p1j1c/tabs_vs_spaces_vs_both/)).
```

Reply:
This feels better to me, slightly more accessible but definitely makes a difference!